pom.xml

```diff
@@ -1815,6 +1827,47 @@ under the License.
 					</configuration>
 				</plugin>
 
+				<!-- Due to the Flink build setup, "mvn spotless:apply" and "mvn spotless:check"
+				don't work. You have to use the fully qualified name, i.e.
+				"mvn com.diffplug.spotless:spotless-maven-plugin:apply" -->
```

Comment:
@zentol said that this can happen quite a lot. On a fresh Maven project `mvn spotless:apply` works, but not for us. We're stretching Maven quite a bit and that shows more and more. 🙈